padle/padle.py

```diff
@@ -0,0 +1,489 @@
+
+    @padle.command(aliases=["sub"])
+    async def subscribe(self, ctx, sub_arg: bool = True):
+        """Subscribe to daily notifications of new PADles"""
+        prefix = ctx.prefix
+        subbed_users = await self.config.subs()
+        if not sub_arg:
+            if ctx.author.id in subbed_users:
+                return await self.unsubscribe(ctx)
+            return await send_cancellation_message(ctx, "You are not subscribed.")
+        if ctx.author.id not in subbed_users:
+            async with self.config.subs() as subs:
+                subs.append(ctx.author.id)
+            return await send_confirmation_message(ctx, "You will now receive notifications of new PADles. "
+                                                        "You can unsubscribe with `{}padle subscribe 0`.".format(
+                prefix))
+
+        confirmation = await get_user_confirmation(ctx, "You are already subscribed. Did you mean to unsubscribe?")
+        if confirmation:
+            await self.unsubscribe(ctx)
+        elif confirmation is None:
+            await send_cancellation_message(ctx, "Confirmation timeout")
+        else:
+            await ctx.send("No changes were made, you will still receive notifications of new PADles.")
+
+    async def unsubscribe(self, ctx):
+        async with self.config.subs() as subs:
+            subs.remove(ctx.author.id)
+        return await send_confirmation_message(ctx, "You will no longer receive notifications of new PADles.")
+
+    async def can_play_in_guild(self, ctx):
+        return (ctx.guild is not None and ctx.author.guild_permissions.administrator and ctx.guild.member_count < 10 and
+                await self.config.guild(ctx.guild).allow())
+
+    @padle.command()
+    async def start(self, ctx):
+        """Start a game of PADle"""
+        if await self.config.user(ctx.author).done():
+            return await ctx.send("You have already finished today's PADle!")
+        if await self.config.user(ctx.author).start():
+            return await ctx.send("You have already started a game!")
+        if (ctx.guild is not None and ctx.author.guild_permissions.administrator and ctx.guild.member_count < 10 and
+                not await self.config.guild(ctx.guild).allow()):
+            guild_confirmation = await get_user_confirmation(ctx, "It looks like you're in a private server. "
+                                                                  "Would you like to play PADle in this server"
+                                                                  " rather than in DMs?", timeout=30,
+                                                             force_delete=False, show_feedback=True)
+            if guild_confirmation is None:
+                return await send_cancellation_message(ctx, "Confirmation timeout.")
+            if await self.config.guild(ctx.guild).allow():
+                return await ctx.send("You have already enabled PADle in this server!")
```

Comment:
umm this case shouldn't be possible right? if they've already enabled padle in that server then we should always just let them play, without making them re-confirm?

Reply:
edge case if someone sends two `^padle start` messages and tries to accept both of them